### DISCUSSION TITLE
PORT issue

### DISCUSSION THREAD

**Discussion by joeogro** [cncjs/cncjs#770] [General]
Hi, my first time using cncjs, my machine is powered up and the usb cable is plugged, i already installed the driver, but the cncjs isnt reading any Port of my pc, what should i do?

thanks

**Comment by MitchBradley** [comment]
Check to see if the COM port shows up in Device Manager.  If not you might have to install a device driver for the USB to Serial chip. You should also try connecting to the controller with a serial terminal program. If a simple terminal program cannot connect to it, cncjs will not be able to connect.
